Source/DafnyTestGeneration.Test/AuditorTests.cs

```diff
@@ -0,0 +1,266 @@
+// Copyright by the contributors to the Dafny Project
+// SPDX-License-Identifier: MIT
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Dafny;
+using Xunit;
+
+namespace DafnyTestGeneration.Test;
+
+public class AuditorTests : Setup {
```

Comment:
I might call this `WarningTests` or something, to avoid confusion with the separate feature that uses the name "auditor".

Reply:
It's great to have lots of tests of this, BTW. :)

Reply:
Done!